src/lib/parquet/src/Flow/Parquet/Writer.php

```diff
@@ -30,7 +30,7 @@ public function write(string $path, Schema $schema, iterable $rows) : void
             \unlink($path);
         }
 
-        $stream = \fopen($path, 'wb');
+        $stream = \fopen($path, 'w');
```

Comment:
I think this should stay wb

Reply:
Restored, but it's now added elsewhere as it's not configurable other way... https://cs.symfony.com/doc/rules/function_notation/fopen_flags.html